spec/requests/api/v0/artists/create_artist_request_spec.rb

```diff
@@ -0,0 +1,56 @@
+require "rails_helper"
+
+RSpec.describe "endpoint post api/v0/artists" do
+  describe "As a User" do
+
+    before do
+      @headers = {
+        "CONTENT_TYPE": "application/json",
+        "ACCEPT": "application/json"
+      }
+
+      create_list(:artist, 5)
+    end
+
+    it "can create a artist from the database via HTTP request" do
+      expect(Artist.all.count).to eq(5)
+
+      artist_params = {
+        name: "Ted Lasso",
+        location: "Pacific Ocean",
+        email: "[tatart@gmail.com](mailto:tatart@gmail.com)",
+        password: "FakerPassowrd1"
+      }
+
+      post "/api/v0/artists", params: JSON.generate({ artist: artist_params }), headers: @headers
+      artist = JSON.parse(response.body, symbolize_names: true)[:data]
+
+      expect(Artist.all.count).to eq(6)
+      check_hash_structure(artist, :id, String)
+      check_hash_structure(artist, :attributes, Hash)
+      check_hash_structure(artist[:attributes], :name, String)
+      check_hash_structure(artist[:attributes], :location, String)
+      check_hash_structure(artist[:attributes], :email, String)
+    end
+
+    describe "sad path" do 
+      it "errors when not all atrributes are filled in" do 
+        artist_params = {
+          name: "",
+          location: "Pacific Ocean",
+          email: "",
+          password: "FakerPassowrd1"
```

Comment:
Interesting. I think here is where I was running into the issue of the "Password can't be blank". I think with how this is currently working, it would force you to enter your password into the edit form/change your password any time you edit other attributes, which I don't think we want to do. But I think this is a good question to leave for the check-in on Thursday to see what the best way to handle this might be, or if we're on the right track with how it's being done for Users currently.